src/Map/index.tsx

```diff
@@ -38,6 +38,7 @@ export interface PointOptions {
   pointStrokeColor?: string
   onPointClick?: (feature: MapboxGeoJSONFeature) => void
   onClickZoomIn?: number
+  zoomLocation?: [number, number]
```

Comment:
is this conceptually part of the point options? Feels to me like this might be better as a top level prop?